Makefile

```diff
@@ -55,14 +56,23 @@ clean:
 build/llvm.BUILT:
 	mkdir -p build/llvm
 	cd build/llvm && cmake -G Ninja \
+		-DCMAKE_MODULE_PATH=$(ROOT_DIR)/cmake \
+		-DWASI_SDK_PREFIX=$(WASI_SDK) \
+		-DCMAKE_TOOLCHAIN_FILE=$(ROOT_DIR)/wasi-sdk.cmake \
+		-DLLVM_HOST_TRIPLE=wasm32-wasi \
+		-DLLVM_ENABLE_THREADS=OFF \
 		-DCMAKE_BUILD_TYPE=MinSizeRel \
 		-DLLVM_ENABLE_TERMINFO=OFF \
 		-DLLVM_ENABLE_ZLIB=OFF \
 		-DLLVM_ENABLE_ZSTD=OFF \
 		-DLLVM_STATIC_LINK_CXX_STDLIB=ON \
 		-DLLVM_HAVE_LIBXAR=OFF \
-		-DCMAKE_OSX_ARCHITECTURES="arm64;x86_64" \
-		-DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 \
+		-DCLANG_ENABLE_ARCMT=OFF \
+		-DCLANG_INCLUDE_TESTS=OFF \
+		-DLLVM_INCLUDE_TESTS=OFF \
+		-DLLVM_INCLUDE_UTILS=OFF \
+		-DLLVM_INCLUDE_BENCHMARKS=OFF \
+		-DLLVM_INCLUDE_EXAMPLES=OFF \
```

Comment:
I think we can/should probably land this hunk separately and without delay since I think it would speed up our builds.

Reply:
I wasn't sure if these things were needed. I'll have a look how much more work it is to support e.g. tests, that would help with validating the changes.

Reply:
They are not needed as far as I know, we don't use any of those binaries.

Reply:
Can you rebase this now that #309 has landed?